.env

```diff
@@ -1 +1,5 @@
-NODE_LOCAL_PORT=3000
\ No newline at end of file
+NODE_LOCAL_PORT=5000
```

Comment:
Is this env variable NODE_LOCAL_PORT not being used anymore? Then it can be removed from this file.